main/webapp/modules/core/langs/translation-en.json

```
@@ -90,6 +91,10 @@
     "core-index-import/this-computer": "This Computer",
     "core-index-import/warning-data-file": "You must specify a data file to import.",
     "core-index-import/warning-import-file": "You must specify a project file.",
+    "core-index-import/warning-import-url": "You must specify a correct url.",
+    "core-index-import/warning-import-input": "You must specify either a project file / url.",
+    "core-index-import/warning-import-two-input": "You must specify only one input field.",
+    "core-index-import/delete-import-file": "Delete file",
```

I think less confusing would be to say **Clear file**
```suggestion
    "core-index-import/delete-import-file": "Clear file",
```